growth/too/models.py

```diff
@@ -145,8 +145,12 @@ def create_all():
             reference_images = {}
             reference_mags = {}
 
-        tessfile = pkg_resources.resource_stream(__name__,
-                                                 'input/%s.tess' % tele)
+        tessfile = os.path.join(app.instance_path, '%s.tess' % tele)
```

Comment:
Please use [`app.open_resource`](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Flask.open_resource) so that you don't have to calculate the path explicitly.